doc-deploy-changelog/action.yml

```
@@ -73,21 +73,52 @@ inputs:
     default: true
     type: boolean
 
-  update-release-branch:
+  release-from-main:
     description: |
-      Whether or not to update the CHANGELOG in the release branch.
-      If ``true``, the branch used to update the CHANGELOG is "release/major.minor".
-      If ``false``, the branch used to update the CHANGELOG is
-      "maint/changelog-update-tag_version" and will be deleted after use.
-    required: false
-    default: true
-    type: boolean
+      Pushing a release tag results in this action running twice:
 
-  update-main-branch:
-    description: |
-      Whether or not to update the CHANGELOG in the main branch.
-      If ``true``, the branch used to update the CHANGELOG is main.
-      If ``false``, check the value of the update-release-branch input.
+      - Once to run ``towncrier`` and consolidate the changelog fragments into a
+      single update of the CHANGELOG file.
+      - Once to release the package after the CHANGELOG file has been updated.
+
+      There are two options when making a release for your repository:
+
+      1. You pushed a tag from a release branch
```

Sorry, I just realized in re-reading this again, it might make sense to be really explicit in the description by specifying which value corresponds to which sequence. Again, maybe going over the top, but at least it's hard to get wrong!
 
```suggestion
      1. You pushed a tag from a release branch (release-from-main is ``false``)
```

---

That's okay! I'm happy to add that in to make it as clear as possible